contracts/LpSugar.vy

```diff
@@ -640,15 +673,28 @@ def _byDataCL(_data: address[4], _token0: address, _token1: address, \
     fee_voting_reward = gauge.feesVotingReward()
     emissions_token = gauge.rewardToken()
 
-  if gauge_alive:
-    emissions = gauge.rewardRate()
+    # fetch total staked liquidity from the ticks surrounding current tick
```

Comment:
I think we can just use this, no?
https://github.com/velodrome-finance/slipstream/blob/main/contracts/core/interfaces/pool/ICLPoolState.sol#L78-L80